src/auro-datepicker.js

```diff
@@ -649,16 +694,17 @@ export class AuroDatePicker extends LitElement {
   // function that renders the HTML and CSS into  the scope of the component
   render() {
     return html`
-      <div>
+      <div part="datepickerContainer">
```

Comment:
What does this allow a user to style that they can't by using auro-dropdown part?

Reply:
Fair point, this is removed.